corehq/apps/app_manager/app_schemas/case_properties.py

```diff
@@ -543,3 +543,10 @@ def _get_usercase_default_properties(domain):
 
     fields_def = CustomDataFieldsDefinition.get_or_create(domain, CUSTOM_USER_DATA_FIELD_TYPE)
     return [f.slug for f in fields_def.get_fields()]
+
+
+def expire_case_properties_caches(domain):
```

Comment:
```suggestion
def expire_case_properties_caches(domain):
    """
    Clears ``all_case_properties_by_domain()`` quickcache for ``domain``
    and all values of params ``include_parent_properties`` and ``exclude_deprecated_properties``.
    """
```

Reply:
Thanks for the suggestion @kaapstorm 

That detail about passing all combinations of params is something to do with how quickcache works, which I agree isn't obvious, so adding a comment sounds right to me though I am not sure it should in the function documentation.

I would image documentation should be something like "Expire any cache related to case properties"?
Even though, for now, it is limited to one function, It's a good placeholder to add any other such caches and expire them all together at one place.

What do you think?

Reply:
Yeah, a comment is fine. Using keyword arguments would also be fine.

I'm not sure about the next developer, but speaking for myself, I understood that we are clearing the cache for all combinations of the function's parameters. I just didn't know what those parameters were, and therefore why they needed to be cleared, without going to look them up.

Reply:
Makes sense about kwargs.

https://github.com/dimagi/commcare-hq/pull/34154/commits/055a014e090dde959c91a47ea13ab7704fba9f1f